content/swiftui/concepts/viewmodifier/terms/textFieldStyle/textFieldStyle.md

```diff
@@ -0,0 +1,130 @@
+---
+Title: '.textFieldStyle()'
+Description: 'Applies a specified style to a TextField.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI ViewModifiers'
+  - 'Xcode'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.textFieldStyle()`** modifier applies a specified style to a [`TextField`](https://www.codecademy.com/resources/docs/swiftui/views/textfield).
+
+## Syntax
+
+```pseudo
+        TextField("label", text: $bindingValue)
+            .textFieldStyle(style)
+```
+
+The style passed as an argument to `.textFieldStyle()` is applied to the `TextField`. Built-in styles are available in Swift UI and it's also possible to create custom styles.
+
+> **Note:** If `.textFieldStyle()` is called on a [`View`](https://www.codecademy.com/resources/docs/swiftui/views), the specified style will be applied to every `TextField` within that view, unless it has its own `.textFieldStyle()`.
+
+## Built-in `TextField` Styles
```

Comment:
This might be better as a bulleted list than as a separate section with sub-headers.

Reply:
@SSwiniarski I changed this to a bulleted list, thanks for the suggestion!